src/VisualStudio/Setup/VisualStudioSetup.csproj

```diff
@@ -147,6 +147,32 @@
     <NuGetPackageToIncludeInVsix Include="Microsoft.Composition" />
     <NuGetPackageToIncludeInVsix Include="System.Reflection.Metadata" />
     <NuGetPackageToIncludeInVsix Include="System.Collections.Immutable" />
+    <NuGetPackageToIncludeInVsix Include="System.Collections" />
```

Comment:
Before we do that we should stop inserting them into ExternalAPI/Roslyn and instead insert each facade into a separate ExternalAPI/{facade-name} directory.

In addition we need to add appropriate binding redirect entries.